src/Slices/Resource/UI/ResourcesPage/ResourcesTable.tsx

```diff
@@ -65,7 +65,10 @@ export const ResourcesTable: React.FC<Props> = ({
   return (
     <TableComposable {...props} variant={TableVariant.compact}>
       <Thead>
-        <Tr>{heads}</Tr>
+        <Tr>
+          <Th></Th>
```

Comment:
why the empty tag?

Reply:
without it toggle cell move everything out of it's column

Reply:
Is this like a known issue? Is it used in other places too?

Reply:
I wouldn't say it is an issue, more like how table work, the same solution is used in service inventory, other implementation use custom Toggle button with additional spanon header

Reply:
ok!